app/src/main/java/com/github/se/wanderpals/model/viewmodel/SessionViewModel.kt

```diff
@@ -23,14 +25,15 @@ class SessionViewModel(private val tripsRepository: TripsRepository) : ViewModel
     }
   }
 
-  fun updateUserForCurrentUser(tripId: String) {
+  fun updateUserForCurrentUser(tripId: String, navigationActions: NavigationActions) {
```

Comment:
The method is not documented and was already present before your modifications. You can probably add the documentation for it even if it was not initally your job.